spec/models/event_spec.rb

```diff
@@ -1,8 +1,168 @@
 require 'spec_helper'
 
 describe Event do
+  subject { create(:event) }
+  let(:conference) { create(:conference) }
+  let(:event) { create(:event, program: conference.program) }
+  let(:user) { create(:user) }
 
-  describe 'abstract_word_count' do
+  describe 'association' do
+    it 'should belong to program' do
+      should belong_to :program
```

Comment:
Please avoid using the deprecated ``should`` syntax
http://betterspecs.org/#expect